app/ai/voice/agents/breeze_buddy/template/transition.py

```
@@ -58,6 +59,40 @@ async def transition_handler(
+            function_name=function_name or "unknown",
+            arguments=args,
+        )
+        logger.info(f"HITL DEBUG transition.py: HITL result={confirmation}")
+
+        if not confirmation.get("approved"):
+            logger.info(
+                f"HITL rejected for {function_name}: {confirmation.get('reason')}"
+            )
+            return {
+                "status": "rejected",
+                "error": f"Function execution rejected",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)